src/renderer/component/categoryList/view.jsx

```diff
@@ -250,38 +251,30 @@ class CategoryList extends PureComponent<Props, State> {
             )}
           </div>
           {showScrollButtons && (
-            <div
-              className={classnames('card-row__scroll-btns', {
-                'card-row__scroll-btns--sub-component': isSubComponent,
-              })}
-            >
+            <nav className="media-group__header-navigation">
               <Button
-                className="btn--arrow"
                 disabled={!canScrollPrevious}
                 onClick={this.handleScrollPrevious}
                 icon={ICONS.ARROW_LEFT}
               />
               <Button
-                className="btn--arrow"
                 disabled={!canScrollNext}
                 onClick={this.handleScrollNext}
                 icon={ICONS.ARROW_RIGHT}
               />
-            </div>
+            </nav>
           )}
-        </div>
+        </header>
         {obscureNsfw && isCommunityTopBids ? (
-          <div className="card-row__message help">
+          <p className="media__message media__message--help">
             {__(
               'The community top bids section is only visible if you allow mature content in the app. You can change your content viewing preferences'
             )}{' '}
             <Button button="link" navigate="/settings" label={__('here')} />.
-          </div>
+          </p>
         ) : (
-          <div
-            className={classnames('card-row__scrollhouse', {
-              'card-row__scrollhouse--sub-component': isSubComponent,
-            })}
+          <ul
+            className="media-scrollhouse"
```

Comment:
There used to be a min-height on this for cases like the empty cards on line 307. Currently suggested subscriptions don't render any cards until something is loaded because there is no height. Maybe the placeholders just need a height?